Add to readme - Starting with Rails section (#4954)
plataformatec/devise: README.md

## Starting with Rails?

> If you are building your first Rails application, we recommend you *do not* use Devise. Devise requires a good understanding of the Rails Framework. In such cases, we advise you to start a simple authentication system from scratch. Click on the links below to a few sources to help you get started.

Reviewer: ```suggestion
If you are building your first Rails application, we recommend you *do not* use Devise. Devise requires a good understanding of the Rails Framework. In such cases, we advise you to start a simple authentication system from scratch. Click on the links below to a find a few resources that will help you get started:
```

* [Michael Hartl's online book](https://www.railstutorial.org/book/modeling_users)

> 		- detailed information on User models and databases

Reviewer: ```suggestion
		- Detailed information on User models and databases
```

* [Ryan Bates' Railscast](http://railscasts.com/episodes/250-authentication-from-scratch)
		- Example codes for a basic user and password authentication
* [Codecademy's](http://www.codecademy.com/en/learn/rails-auth)
		- Interactive tutorial on Ruby on Rails: Authentication and Authorization

> Once you have solidified your understanding of Rails and authentication mechanisms, checkout the resource below on creating rails application with Devise.

Reviewer: ```suggestion
Once you have solidified your understanding of Rails and authentication mechanisms, checkout the resource below to learn how to create a Rails application with Devise:
```

* [FreeCodeCamp](https://medium.freecodecamp.org/lets-create-an-intermediate-level-ruby-on-rails-application-d7c6e997c63f)

> 		- Step by step rails tutorial in creating a full rails application using Devise gem for authentication

Reviewer: ```suggestion
		- Tutorial: how to create a Rails application using Devise for authentication
```
